.github/CODEOWNERS

```diff
@@ -24,3 +24,4 @@
 /charts/prometheus-redis-exporter/ @acondrat @zanhsieh
 /charts/prometheus-snmp-exporter/ @miouge1
 /charts/prometheus-to-sd/ @acondrat
+/charts/prometheus-kafka-exporter/ @gkarthiks
```

Comment:
Can you move this up to be in alphabetical order? Or just run https://github.com/prometheus-community/helm-charts/blob/main/scripts/check-codeowners.sh 😄